### PR TITLE
Update tutorial.md

### PR DESCRIPTION
**Pull request by Expertium**

### CODE REVIEW

**Comment by Expertium** [comment]
@user1823 check if everything is good

File: docs/tutorial.md [hunk 1]
```diff
@@ -120,68 +120,86 @@ Since the tool is experimental, it is better to use your intuition to come up wi
...
 
-Q10: What will happen if I review my cards on a device where FSRS is not supported (or disabled) and then on another device where FSRS is enabled?
+Q12: What will happen if I review my cards on a device where FSRS is not supported (or disabled) and then on another device where FSRS is enabled?
 
-A10: Your intervals will become inaccurate, but it won't corrupt your cards and make them unusable. It will just make FSRS bad at what it's supposed to do: maintain your retention at a specified level.
+A12: Your intervals will become inaccurate, but it won't corrupt your cards and make them unusable. It will just make FSRS bad at what it's supposed to do: maintain your retention at a specified level.
 
 ***
 
 You haven't seen the answer for your question? Here are many questions asked by others: https://github.com/open-spaced-repetition/fsrs4anki/issues?q=is%3Aissue+label%3Aquestion+
```
Comment (user1823) [inline docs/tutorial.md:201]:
```suggestion
Didn't find the answer to your question? You can browse through several other questions asked by users: https://github.com/open-spaced-repetition/fsrs4anki/issues?q=is%3Aissue+label%3Aquestion+
```

File: docs/tutorial.md [hunk 2]
```diff
@@ -120,68 +120,86 @@ Since the tool is experimental, it is better to use your intuition to come up wi
...
 
 If you still have trouble, please open a new issue to provide the details about that.
 
-https://github.com/open-spaced-repetition/fsrs4anki/issues/new/choose
\ No newline at end of file
+https://github.com/open-spaced-repetition/fsrs4anki/issues/new/choose
```
Comment (user1823) [inline docs/tutorial.md:205]:
```suggestion
Is your issue still unsolved? Please open a new issue to provide the details: https://github.com/open-spaced-repetition/fsrs4anki/issues/new/choose
```

File: docs/tutorial.md [hunk 3]
```diff
@@ -120,68 +120,86 @@ Since the tool is experimental, it is better to use your intuition to come up wi
...
-A3: FSRS only takes into account one review per day. If you review a card multiple times per day, only the chronologically first review will be used by the optimizer.
+A3: Yes, just turn FSRS off. However, this will not revert the intervals back.
 
 ***
 
-Q4: My first interval for "Easy" is too long! Is this normal?
+Q4: I'm sure I have >1000 reviews, yet when I try to optimize parameters for my preset, I get an error telling me that I don't have enough reviews. Is that a bug?
 
-A4: Yes. Anki tends to give very short first intervals. Don't be surprised if your first interval for "Good" is 5-7 days and your first interval for "Easy" is several weeks long.
+A4: FSRS only takes into account one review per day. If you review a card multiple times per day, only the chronologically first review will be used by the optimizer. Also, if your deck has many subdecks, be sure to check that the preset is applied to the subdecks as well, not just to the parent deck.
```
Comment (user1823) [inline docs/tutorial.md:143]:
```suggestion
A4: FSRS only takes into account one review per day. If you review a card multiple times per day, only the chronologically first review will be used by the optimizer. Also, if your deck has many subdecks, ensure that the preset is also applied to the subdecks, not just to the parent deck.
```

**Review by user1823 — CHANGES_REQUESTED** [review]

**Review by L-M-Sherlock — APPROVED** [review]

**Pull request merged**